.github/workflows/build_msi_installer.yml

```diff
@@ -61,7 +61,7 @@ jobs:
       - name: Install stable promptflow
         if: ${{ github.event.inputs.version != null && github.event.inputs.version != '' }}
         run: |
-          pip install promptflow[azure,executable]==$env:INPUT_VERSION promptflow-tools
+          pip install promptflow[azure,executable,pfs]==$env:INPUT_VERSION promptflow-tools
```

Comment:
should we also update step_sdk_setup action when build dev branch?

Reply:
it has been updated in the previous PR